@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -873,9 +884,22 @@ export class TiltViewClipPlanesEvaluator extends TopViewClipPlanesEvaluator {
         }
         viewRanges.far = farPlane;
 
+        // Compute the focus point (target) distance for current camera and projection setup,
+        // in a same way the MapView component does.
+        // TODO: Target distance does not need to be cached when we provide full support for
+        // target point within all camera tilt angles.
+        const target = MapViewUtils.getWorldTargetFromCamera(camera, projection);
+        if (target !== null) {
+            this.m_targetDistance = camera.position.distanceTo(target);
+        }
+        // Fallback to camera altitude if target not found and not yet set.
+        // NOTE: May be inaccurate when launching map with camera looking to void.
+        else if (this.m_targetDistance < cameraAltitude) {
```

Comment:
If `this.m_targetDistance` is  undefined this will always be false

Reply:
ClipPlanesEvaluator.m_targetDistance is always defined, but may be equal zero, then (unfortunately) we need some fallback, here it is camera distance to ground, I could implement better fallback here, but I guess it is worthless because it is MapViewUtils.getWorldTargetFromCamera() which needs to be improved - it should always return some point - with fallback if required.

Reply:
Okay, either `MapViewUtils.getWorldTargetFromCamera()` should always return sth. or we should always have a valid `targetDistance` in the `ViewState`.
Since the goal is to reduce MapViewUtils and maybe eventually remove it I would go for the latter, but as mentioned in the comment below I will do this later.